src/components/my-career/MyCareerTab.jsx

```diff
@@ -1,42 +1,67 @@
-import React, { useContext } from 'react';
+import React, {
+  useContext, useMemo, useState, useEffect,
+} from 'react';
 
 import { AppContext, ErrorPage } from '@edx/frontend-platform/react';
 import { SearchData } from '@edx/frontend-enterprise-catalog-search';
-import { useLearnerSkillQuiz } from './data/hooks';
+import { useLearnerProfileData } from './data/hooks';
 import { LoadingSpinner } from '../loading-spinner';
 import AddJobRole from './AddJobRole';
 import VisualizeCareer from './VisualizeCareer';
-import { getSkillQuiz } from './data/utils';
-import { SEARCH_FACET_FILTERS } from '../search/constants';
+import { extractCurrentJobID } from './data/utils';
 
 const MyCareerTab = () => {
   const { authenticatedUser } = useContext(AppContext);
   const { username } = authenticatedUser;
 
-  const [learnerSkillQuiz, learnerSkillQuizFetchError] = useLearnerSkillQuiz(
+  const [learnerProfileData, learnerProfileDataFetchError] = useLearnerProfileData(
     username,
   );
+  const [learnerProfileState, setLearnerProfileState] = useState();
```

Comment:
Do we need to define the default state?

Reply:
Default state would be `undefined` here which I don't think i need to explicitly define here.